pkg/common/common.go

```diff
@@ -151,4 +151,55 @@ const (
 	VddkConfigMap = "v2v-vmware"
 	// VddkConfigDataKey is the name of the ConfigMap key of the VDDK image reference
 	VddkConfigDataKey = "vddk-init-image"
+
```

Comment:
curious why this stuff was moved?

Reply:
This was the easiest way I could find to break a chain of dependencies that was causing libnbd to get linked in to pretty much every other binary. I think the problem was that pkg/uploadserver depends on pkg/importer, and a few other packages depend on pkg/uploadserver for some of the constants I moved. The end result was that any binary that used those constants had to link to libnbd to satisfy the indirect dependency on pkg/importer, even tests.test.

There are probably other ways to break up the dependencies so this doesn't happen, but this was the least disruptive option I found so far.

Reply:
cool thanks for the info